src/index.css

```diff
@@ -5,3 +5,7 @@
 .active {
   font-weight: bold;
 }
+
+.input {
+  @apply border-2 border-blue-300 rounded-full px-4 py-2;
+}
```

Comment:
Great use of @apply

Reply:
I agree! what an elegant approach using **@apply**. Your implementation is beautifully clear 👏